xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -203,48 +197,47 @@ bool CGUIDialogPVRGuideInfo::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
-void CGUIDialogPVRGuideInfo::SetProgInfo(const CFileItem *item)
+void CGUIDialogPVRGuideInfo::SetProgInfo(const EPG::CEpgInfoTagPtr &tag)
 {
-  *m_progItem = *item;
+  m_progItem = tag;
```

Comment:
Yes, I'm sure this is correct. Before, a copy of a FileItem holding a globally shared tag was held. Now, the globally shared tag is held directly.